pkgs/development/libraries/libint/default.nix

```
@@ -1,6 +1,6 @@
 { lib, stdenv, fetchFromGitHub, autoconf, automake, libtool
-, python3, perl, gmpxx, mpfr, boost, eigen, gfortran
-, enableFMA ? false
+, python3, perl, gmpxx, mpfr, boost, eigen, gfortran, cmake, ninja
+, enableFMA ? false, enableAVX ? false
```

```suggestion
, enableFMA ? false, enableAVX ? stdenv.hostPlatform.avxSupport
```
is maybe a better default ?

---

Cool, didn't know that this exists. :partying_face: How exactly would this behave? Say if hydra builds this and the hydra worker has AVX enabled, would it go into the cache like this? This again would break this package for a lot of hardware.

---

No this flags will only be true if the user declaratively sets `localSystem` to a specific architecture/cpu model: https://nixos.wiki/wiki/Build_flags

---

Thank you for pointing this out :slightly_smiling_face: